edenai_apis/features/ocr/ocr_interface.py

```
@@ -141,6 +141,27 @@ def ocr__financial_parser(
         """
         raise NotImplementedError
 
+    @abstractmethod
+    async def ocr__afinancial_parser(
```

Pylint (4.0.3)

[refactor] Too many positional arguments (6/5)

(R0917)